src/Components/Forms/AuthForms/SignUpForm.tsx

```diff
@@ -1,88 +1,88 @@
+'use client';
 import { useEffect, useState } from 'react';
 import { useRouter } from 'next/navigation';
 import Image from 'next/image';
 import Link from 'next/link';
 import { useFormik, FormikHelpers } from 'formik';
 //@ts-ignore
 import { Button, Input } from 'technogetic-iron-smart-ui';
-
 import { SignupSchema } from '@/utils/schema';
-import { SignupFormValuesType } from '../authInterfaces';
 import { signUpService } from '@/services/authServices';
 import styles from '@/styles/auth.module.scss';
+import { SignupFormValuesType } from '@/Components/pageComponents/auth/authInterfaces';
+import { DecodedToken, decodeJWt } from '@/utils/globalfunctions';
+import { toast } from 'react-toastify';
 
-const SignupForm = () => {
-  const [isLoading, setIsLoading] = useState<boolean>(false);
-  const [rememberMe] = useState<boolean>(false);
-  const [error, setError] = useState<string>('');
+const initialValues: SignupFormValuesType = {
+  fullName: '',
+  email: '',
+  password: '',
+};
 
+export const SignupForm = () => {
   const router = useRouter();
+  const [error, setError] = useState<string>('');
 
-  const initialValues: SignupFormValuesType = {
-    fullName: '',
-    userName: '',
-    email: '',
-    password: '',
-    upiId: '',
+  const handleRedirect = (token: string) => {
```

Comment:
if this function used in multiple files, create a common function and use it.

Reply:
done.